wiki/Community/Forum/en.md

```diff
@@ -0,0 +1,54 @@
+---
+tags:
+  - forums
+  - subforum
+---
+
+# Forum
+
+*See also: [Internet forum](https://en.wikipedia.org/wiki/Internet_forum)*
+
+[The osu! **forums**](https://osu.ppy.sh/community/forums) are a place for conversations where messages are grouped by subjects, and no quick response is expected. The discussions are searchable, stored forever, and may be revisited at any time, which makes them different from the near-instant messages in [osu! chat](/wiki/Community/Internet_Relay_Chat). As a result, forum messages are longer, contain more information, and typically provide more details than chat messages would.
+
+The osu! forums provide features similar to most internet forums: [BBCode](/wiki/BBCode) markup, polls, pins and subscriptions, private messages, and [moderation](#moderation).
+
+## Structure
+
+Each of the osu! forums contains a variety of **topics**, sometimes called threads, as well as smaller categories, or **subforums**. Most of the forums are English only, but there also exists a number of [language-specific](https://osu.ppy.sh/community/forums#forum-23) forums for local communities that are large enough.
+
+While some forums, such as [Mapping Discussion](https://osu.ppy.sh/community/forums/56), have specific requirements or guidelines, all of them follow [the community rules](/wiki/Rules).
+
+## Roles and user groups
+
+Some users may belong to one or several custom groups, based on their community tasks, access rights, or past activity. The list of user groups can be found at [The Team § User groups](/wiki/People/The_Team#user-groups).
+
+In addition to profile badges provided by group membership, users may have [custom forum titles](/wiki/People/Users_with_unique_titles) displayed under the avatar in each of their posts.
```

Comment:
I believe I understand [what @cl8n is saying](https://github.com/ppy/osu-wiki/pull/6454#discussion_r763726016), but from a reader's perspective, it makes sense to explain this since it is something that people encounter in the forums. Yes, and beatmap discussions and yes, in profiles as well, but they deserve a mention here (with links to the proper articles) since it may not be obvious for first-timers when looking at a forum post what the things below the avatar mean. In other words, having this here contributes to connectivity of the wiki

Can remove "profile badge" though since that those aren't shown in posts, and should also probably link to `User_title` instead 

```suggestion
Some users may belong to one or several custom groups, based on their community tasks, access rights, or past activity. A full list of user groups can be found at [The Team § User groups](/wiki/People/The_Team#user-groups).

In addition to user group badges provided by group membership, users may have [custom user titles](/wiki/en/User_title) displayed under the avatar in each of their posts.
```

Reply:
*The link is wrong in the suggestion, it should be `(/wiki/User_title)` instead of `(/wiki/en/User_title)`.*

Reply:
👍